src/App.tsx

```diff
@@ -10,33 +11,27 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
+  const onTabSelected = (tab: {
```

Comment:
create separate interface for tab. And i guess is function isn't needed, you can use directly setSelectedTab